examples/notebooks/FastaiLRFinder_MNIST.ipynb

```diff
@@ -238,11 +255,36 @@
     "id": "NcT19wqkmvd6"
    },
    "source": [
-    "Let's now setup suggested learning rate to the optimizer and we can train the model with optimal learning rate.\n",
+    "Let's now apply suggested learning rate to the optimizer and create a new trainer, then we can train the model with optimal learning rate.\n",
     "\n",
     "*Note, that model and optimizer were restored to their initial states when `FastaiLRFinder` finished.*"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "lr_finder.apply_suggested_lr(optimizer)\n",
+    "print(optimizer.param_groups[0]['lr'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# We can reset the parameters of our model\n",
+    "# or we can just create new instance for the model\n",
+    "new_model = Net()\n",
+    "new_model.to(device)\n",
```

Comment:
If you use new model, you have to set parameters to the optimizer as it has old model's params right now